protocol/election-module/cannonfile.toml

```diff
@@ -0,0 +1,125 @@
+name = "synthetix-election-module"
+description = "On-chain voting for synthetix councils"
+version = "<%= package.version %>"
+
+[setting.salt]
+defaultValue = "governance"
+
+# Deployment Owner, defaults to first hardhat account
+[setting.owner]
+defaultValue = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
+
+[setting.council_token_name]
+defaultValue = "Synthetix Governance Module"
+
+[setting.council_token_symbol]
+defaultValue = "SNXGOV"
+
+[setting.init_council_member]
+defaultValue = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266" # can only specify 1
+
+[setting.epoch_start]
+
+[setting.epoch_duration]
+defaultValue = "90" # days
+
+[setting.voting_period_duration]
+defaultValue = "7" # days
+
+[contract.InitialModuleBundle]
+artifact = "InitialModuleBundle"
+create2 = true
+
+[contract.InitialProxy]
```

Comment:
Does this mean that the previous proxies are deprecated (and thus the history of past elections)?

Reply:
becuaes we are using `create2` unless we change the proxy code in the future, we should not need to migrate again.

Reply:
Cool, thanks guys